crates/packaging/src/https.rs

```diff
@@ -405,3 +408,48 @@ impl<R: Read> Read for HashReader<R> {
         Ok(bytes_read)
     }
 }
+
+/// Prints download progress to stdout
+struct ProgressReporter<R: Read> {
+    read: usize,
+    total: Option<usize>,
+    reader: R,
+}
+
+impl<R: Read> ProgressReporter<R> {
+    fn new(reader: R, total: Option<usize>) -> Self {
+        ProgressReporter {
+            read: 0,
+            total,
+            reader,
+        }
+    }
+}
+
+impl<R: Read> Read for ProgressReporter<R> {
+    fn read(&mut self, buf: &mut [u8]) -> std::io::Result<usize> {
+        let size = self.reader.read(buf)?;
+
+        self.read += size;
+
+        if let Some(total) = self.total {
+            print!(
```

Comment:
Oh, one other thing, these should probably be to stderr. so `eprint!`. Cause if you do `roc run app.roc`, you don't want it to mess with the regular output of `app.roc`. Otherwise, PR looks great!

Reply:
Done!